### PR TITLE
8327859: Remove applet usage from JColorChooser tests Test4319113

### PR DESCRIPTION
**Pull request by alisenchung**
Removing applet usage from manual JColorChooser tests

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8327859](https://bugs.openjdk.org/browse/JDK-8327859): Remove applet usage from JColorChooser tests Test4319113 (**Bug** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)
 * [Tejesh R](https://openjdk.org/census#tr) (@TejeshR13 - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18210/head:pull/18210` \
`$ git checkout pull/18210`

Update a local copy of the PR: \
`$ git checkout pull/18210` \
`$ git pull https://git.openjdk.org/jdk.git pull/18210/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18210`

View PR using the GUI difftool: \
`$ git pr show -t 18210`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18210.diff">https://git.openjdk.org/jdk/pull/18210.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18210#issuecomment-1989693164)

### CODE REVIEW

File: test/jdk/javax/swing/JColorChooser/Test4319113.java [hunk 1]
```diff
@@ -50,56 +38,65 @@
...
 
-    @Override
-    public void actionPerformed(ActionEvent actionEvent) {
-        Object object = actionEvent.getSource();
-        Component component = object instanceof Component ? (Component)object : null;
-        JDialog jDialog = JColorChooser.createDialog(component, "ColorChooser", false, new JColorChooser(Color.BLUE), null, null);
-        jDialog.setVisible(true);
-    }
+    private static JFrame test() {
+        JFrame frame = new JFrame("Test4319113");
```
Comment (TejeshR13) [inline test/jdk/javax/swing/JColorChooser/Test4319113.java:65]: Title can be something like "JColorChooser behavior test".

File: test/jdk/javax/swing/JColorChooser/Test4319113.java [hunk 2]
```diff
@@ -50,56 +38,65 @@
...
+ * @library /test/jdk/java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
+ * @summary Tests the open JColorChooser behavior on LaF change.
+ * @run main/manual Test4319113
+ */
+public class Test4319113 {
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test4319113")
+                .instructions("Choose a LaF using the dropdown. Click on the Show JColorChooser button.\n" +
```
Comment (TejeshR13) [inline test/jdk/javax/swing/JColorChooser/Test4319113.java:53]: I feel its better to use a separate String variable for Test instructions and then use it here.

File: test/jdk/javax/swing/JColorChooser/Test4319113.java [hunk 3]
```diff
@@ -50,56 +38,68 @@
...
 
-public class Test4319113
-extends JApplet
-implements ActionListener {
-    private final JFrame frame = new JFrame("frame");
-    private JComboBox cbPlaf;
+/*
+ * @test
+ * @bug 4319113
+ * @library ../../../java/awt/regtesthelpers
```
Comment (prrace) [inline test/jdk/javax/swing/JColorChooser/Test4319113.java:44]: absolute please

**Review by prrace — APPROVED** [review]

**Review by TejeshR13 — APPROVED** [review]

**Pull request closed without merging**